docs/aws/audit/redshiftmonitoring/rules/audit_logging_enabled.mdx

```diff
@@ -23,6 +23,192 @@ HIPAA, NIST, SOC2, NISTCSF, PCIDSS
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console.
+2. Navigate to the Redshift dashboard by selecting "Services" from the top menu, then selecting "Redshift" under the "Database" category.
+3. In the Redshift dashboard, select "Clusters" from the left-hand menu. This will display a list of all your Redshift clusters.
+4. For each cluster, select the cluster name to open its configuration details. Under the "Configuration" tab, look for the "Logging" section. If the "Logging Status" is set to "Disabled", then the Redshift Cluster Audit does not have logging enabled.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by following the instructions provided in the AWS CLI User Guide.
+
+2. List all Redshift clusters: Use the following AWS CLI command to list all the Redshift clusters in your AWS account:
+
+   ```
+   aws redshift describe-clusters --query 'Clusters[*].[ClusterIdentifier]' --output text
+   ```
+   This command will return a list of all Redshift cluster identifiers.
+
+3. Check logging status: For each Redshift cluster, use the following AWS CLI command to check the logging status:
+
+   ```
+   aws redshift describe-logging-status --cluster-identifier <your-cluster-identifier>
+   ```
+   Replace `<your-cluster-identifier>` with the actual identifier of your Redshift cluster. This command will return the logging status of the specified Redshift cluster.
+
+4. Analyze the output: If the `LoggingEnabled` field in the output is set to `false`, it means that logging is not enabled for the Redshift cluster. If it's set to `true`, it means that logging is enabled.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install and configure AWS SDK for Python (Boto3) on your local system. Boto3 allows you to directly create, update, and delete AWS services from your Python scripts.
+
+2. Import the necessary libraries and establish a session with AWS using your access keys.
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'
+)
+```
+
+3. Create a Redshift client using the session created above.
+
+```python
+redshift_client = session.client('redshift')
+```
+
+4. Use the `describe_logging_status` method to check the logging status of the Redshift cluster. This method returns the status of logging for a specified cluster.
+
+```python
+def check_redshift_logging_status(cluster_identifier):
+    response = redshift_client.describe_logging_status(
+        ClusterIdentifier=cluster_identifier
+    )
+    return response['LoggingEnabled']
+
+cluster_identifier = 'your-cluster-identifier'
+logging_enabled = check_redshift_logging_status(cluster_identifier)
+
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>